### PR TITLE
Updated stories and create post components for the desktop version of…

### PR DESCRIPTION
**Pull request by cholakovvv**
… the main page

### CODE REVIEW

**Review by petya0111 — COMMENTED** [review]
Another pull will be created for a desktop main page with rounded corners.

File: client/src/components/Styles/Stories.scss [hunk 1]
```diff
@@ -1,143 +1,138 @@
-.stories-root-box{
-    width: 10%;
-    display: block;
-    text-align: center;
+.add-story-box {
+  height: 10vh;
+  width: 83vw;
+  // margin-top: 7rem;
```
Comment (IvayloSlavchev) [inline client/src/components/Styles/Stories.scss:4]: You can remove this comment

**Review by IvayloSlavchev — APPROVED** [review]
Great job, @SCholakov05!

**Review by di-amond — APPROVED** [review]
Good job!

**Review by petya0111 — APPROVED** [review]
Great work mate :)

**Pull request merged**